src/pages/DataAccessRequestApplication.js

```diff
@@ -54,7 +54,6 @@ class DataAccessRequestApplication extends Component {
         hmb: false,
         poa: false,
         diseases: false,
-        ontologies: [],
```

Comment:
Revert changes to this file, the ontologies referred to here is not the same thing as adding and removing ontology files in GCS, this is for ontology terms that are part of a data access request.